webserver/login/__init__.py

```diff
@@ -33,6 +34,21 @@ def load_user(user_id):
     else:
         return None
 
+@login_manager.request_loader
+def load_user(request):
+    key = request.headers.get("Authorization")
+    user = None
+    if key:
+        parts = key.split(" ")
+        if len(parts) == 2 and parts[0] == "Token":
+            user = db.user.get_by_api_key(parts[1])
+        else:
+            raise Unauthorized
+    if user:
+        return User.from_dbrow(user)
+    else:
+        return None
```

Comment:
This affects authentication outside of the API as well, right? We probably don't want that.